twigextensions/ElixirTwigExtension.php

```diff
@@ -0,0 +1,62 @@
+<?php
+namespace Craft;
+
+use Twig_Extension;
+use Twig_Filter_Method;
```

Comment:
Looks like this class is deprecated in favor of `Twig_SimpleFilter` http://twig.sensiolabs.org/doc/deprecated.html#filters